include/mbedtls/hkdf_tls.h

```diff
@@ -0,0 +1,162 @@
+/**
+* \file hkdf_tls.h
+*
+* \brief TLS 1.3-specific HKDF functionality
```

Comment:
The `\brief` description isn't adequate and the Copyright header needs to be separated out from the `\brief` summary.